Developer docs don't include instructions for running the testing framework locally, and also appear to be missing some setup steps.

I went to run pytest, and start experimenting with mutmut locally, and got this error:

```
(.venv) [esauer 🎩︎pelorus] (master)$ pytest
===================================================================================================== test session starts =====================================================================================================
platform linux -- Python 3.9.4, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/esauer/src/pelorus
plugins: pylava-0.3.0
collected 25 items / 2 errors / 23 selected                                                                                                                                                                                   

=========================================================================================================== ERRORS ============================================================================================================
____________________________________________________________ ERROR collecting samples/basic-nginx/galaxy/openshift-applier/molecule/default/tests/test_applier.py _____________________________________________________________
ImportError while importing test module '/home/esauer/src/pelorus/samples/basic-nginx/galaxy/openshift-applier/molecule/default/tests/test_applier.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
samples/basic-nginx/galaxy/openshift-applier/molecule/default/tests/test_applier.py:3: in <module>
    import testinfra.utils.ansible_runner
E   ModuleNotFoundError: No module named 'testinfra'
____________________________________________________ ERROR collecting samples/nginx-jenkins-binary-fromdir/galaxy/openshift-applier/molecule/default/tests/test_applier.py ____________________________________________________
ImportError while importing test module '/home/esauer/src/pelorus/samples/nginx-jenkins-binary-fromdir/galaxy/openshift-applier/molecule/default/tests/test_applier.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
samples/nginx-jenkins-binary-fromdir/galaxy/openshift-applier/molecule/default/tests/test_applier.py:3: in <module>
    import testinfra.utils.ansible_runner
E   ModuleNotFoundError: No module named 'testinfra'
=================================================================================================== short test summary info ===================================================================================================
ERROR samples/basic-nginx/galaxy/openshift-applier/molecule/default/tests/test_applier.py
ERROR samples/nginx-jenkins-binary-fromdir/galaxy/openshift-applier/molecule/default/tests/test_applier.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
====================================================================================================== 2 errors in 8.86s ======================================================================================================

```

We should update the developer docs to provide the necessary steps to make tests locally runnable as part of development cycle.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,2 +1,5 @@
 [tool.isort]
 profile = "black"
+
+[tool.pytest.ini_options]
+testpaths = ["exporters/tests"]

